.gitignore

```diff
@@ -63,3 +63,6 @@ target/
 # External Libraries
 wger/core/static/bower_components
 node_modules
+
+# Fitbit environment
+.env
```

Comment:
we should add any env variables needed by Fitbit to other app variables.

Reply:
Please shed more light on this. Which other app variables are you referring to?